packages/dd-trace/src/plugins/log_plugin.js

```diff
@@ -43,12 +43,21 @@ module.exports = class LogPlugin extends Plugin {
       this.tracer.inject(span, LOG, holder)
       arg.message = messageProxy(arg.message, holder)
     })
+    this.structured = false // default unstructured logger
+  }
+
+  _injectLogs (config) {
```

Comment:
This is not injecting so the method name is confusing. Maybe something like `_shouldInject`?